sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryBaseEvent.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.protocol.Breadcrumb
+import io.sentry.kotlin.multiplatform.protocol.SentryId
+import io.sentry.kotlin.multiplatform.protocol.User
+
+/** Base class for all Sentry events. */
+public abstract class SentryBaseEvent(public open var eventId: SentryId = SentryId.EMPTY_ID) {
+
+    public open var release: String? = null
+    public open var environment: String? = null
+    public open var platform: String? = null
+    public open var user: User? = null
+    public open var serverName: String? = null
+    public open var dist: String? = null
+
+    /** This is not thread-safe */
+    public var contexts: Map<String, Any>? = mapOf()
+        internal set
+
+    /** This is not thread-safe */
+    public open var breadcrumbs: MutableList<Breadcrumb>? = mutableListOf()
```

Comment:
L: All those lists, maps, etc don't need to be nullable if you are initing during declaration with an empty value.
It's fine as it is, somebody can still set it to `null`, but it'd be natural to just do `breadcrumbs.clear()`.